about.css

```diff
@@ -0,0 +1,207 @@
+/* http://meyerweb.com/eric/tools/css/reset/ 
```

Comment:
Good job splitting the CSS into seperate files for each page.  Another thing that you can do is take the reset/normalize CSS and put it in its own file too.  That way you don't have to copy when creating new CSS files.